chain-10-tree-1.json

```diff
@@ -491,37 +536,51 @@
         "responsibilities": [
           {
             "link": "",
-            "label": "Declare your intentions for how you intend to contribute to Hats Protocol and/or the Hats protoDAO for the coming Season"
+            "label": "Declare seasonal contribution intentions",
```

Comment:
"intentions for how you intend" is awkward

Suggestion:
"Declare your intentions for how you will contribute to Hats Protocol and/or the Hats protoDAO for the coming Season"